onnx/defs/tensor/defs.cc

```diff
@@ -2226,6 +2226,65 @@ ONNX_OPERATOR_SET_SCHEMA(
           resizeShapeInference(ctx, true);
         }));
 
+static const char* GridSampler_ver13_doc = R"DOC(
```

Comment:
nit: GridSampler_ver15_doc

Reply:
It is unclear to me what this function is supposed to compute. Can we expand the description to be more complete, or add a link to some existing function in other frameworks, or both? Thanks.

Reply:
I prefer both as opposed to just adding the link to some existing function in other framework...

Reply:
> It is unclear to me what this function is supposed to compute. Can we expand the description to be more complete, or add a link to some existing function in other frameworks, or both? Thanks.

I only find this op in PyTorch, have posed the link in description. And this op is from Spatial Transformer Networks, I also added the paper link too.